docs/design_document/design.tex

```diff
@@ -69,10 +69,14 @@ \section{Definitions}
 \section{Technology Overview}
 
 \subsection{Operating System}
-An operating system for an intelligent agent serves as the brain of the intelligent agent. The operating system provides the tools necessary to make the robot do an unimaginable number of different things. One of those things that it can help us out with is object recognition and classification. The operating system that we will use for our robot object recognition project is ROS. ROS (Robot Operating System) is the most common operating system for robotic systems. ROS is open source and has BSD licensing which are two big reasons we chose it as part of our design. Being open source is one of the main reasons that ROS has evolved so quickly and become so prominent in the field of robotics research and development. Two key concepts exist that stand out more than the others and that is the package and the stack. The package is a directory that has external libraries, API data, configuration files, nodes and an XML configuration file. The stack is a collection of packages that offer functionalities like navigation, positioning, mapping, and others \cite{ROSintro}.
+Ubuntu is the operating system used on our project to control both the intelligent agent and all computers that communicate with it. The intelligent agent's on-board OS functions like any other computer, but runs a specialized program called ROS (Robot Operating System) to control all "robotic" functions. The computer used to communicate with the intelligent agent must also run Ubuntu and ROS, as the software used alongside ROS must operate through it. Programs such as Rviz, Gmapping, and Mapsaver are all used as auxilary programs to aid us in gathering our test data. 
+
+While ROS is not a true operating system in the same sense Linux is, the program runs constantly in the background of the intelligent agent managing everything from sensors, to motors, to inter-process communication. Compontents of the intelligent agent are registered as devices under ROS and controlled similarly to typical device drivers. 
```

Comment:
I don't think there is a big enough relation between ROS and Linux to say this, ROS is a framewok of tools and software packages for controlling robots while Linux is a kernel which connects GUI applications to the CPU, RAM, I/O, etc.